jimp.d.ts

```diff
@@ -234,211 +234,211 @@ declare namespace Jimp {
         cropQuiet(
             x: number,
             y: number,
             w: number,
             h: number,
-            cb?: Jimp.ImageCallback<T>
+            cb?: Jimp.ImageCallback
         ): this;
 
         // Color methods
-        brightness(val: number, cb?: Jimp.ImageCallback<T>): this;
-        contrast(val: number, cb?: Jimp.ImageCallback<T>): this;
-        posterize(n: number, cb?: Jimp.ImageCallback<T>): this;
-        greyscale(cb?: Jimp.ImageCallback<T>): this;
-        grayscale(cb?: Jimp.ImageCallback<T>): this;
-        opacity(f: number, cb?: Jimp.ImageCallback<T>): this;
-        sepia(cb?: Jimp.ImageCallback<T>): this;
-        fade(f: number, cb?: Jimp.ImageCallback<T>): this;
-        convolution(kernel: number[][], cb?: Jimp.ImageCallback<T>): this;
+        brightness(val: number, cb?: Jimp.ImageCallback): this;
+        contrast(val: number, cb?: Jimp.ImageCallback): this;
+        posterize(n: number, cb?: Jimp.ImageCallback): this;
+        greyscale(cb?: Jimp.ImageCallback): this;
+        grayscale(cb?: Jimp.ImageCallback): this;
+        opacity(f: number, cb?: Jimp.ImageCallback): this;
+        sepia(cb?: Jimp.ImageCallback): this;
+        fade(f: number, cb?: Jimp.ImageCallback): this;
+        convolution(kernel: number[][], cb?: Jimp.ImageCallback): this;
         convolution<T>(
             kernel: number[][],
             edgeHandling: string,
-            cb?: Jimp.ImageCallback<T>
+            cb?: Jimp.ImageCallback
         ): this;
-        opaque(cb?: Jimp.ImageCallback<T>): this;
-        pixelate(size: number, cb?: Jimp.ImageCallback<T>): this;
+        opaque(cb?: Jimp.ImageCallback): this;
+        pixelate(size: number, cb?: Jimp.ImageCallback): this;
         pixelate(
             size: number,
             x: number,
             y: number,
             w: number,
             h: number,
-            cb?: Jimp.ImageCallback<T>
+            cb?: Jimp.ImageCallback
         ): this;
-        convolute(kernel: number[][], cb?: Jimp.ImageCallback<T>): this;
+        convolute(kernel: number[][], cb?: Jimp.ImageCallback): this;
         convolute(
             kernel: number[][],
             x: number,
             y: number,
             w: number,
             h: number,
-            cb?: Jimp.ImageCallback<T>
+            cb?: Jimp.ImageCallback
         ): this;
-        color(actions: ColorAction[], cb?: Jimp.ImageCallback<T>): this;
-        colour(actions: ColorAction[], cb?: Jimp.ImageCallback<T>): this;
+        color(actions: ColorAction[], cb?: Jimp.ImageCallback): this;
+        colour(actions: ColorAction[], cb?: Jimp.ImageCallback): this;
 
         // Shape methods
-        rotate(deg: number, cb?: Jimp.ImageCallback<T>): this;
+        rotate(deg: number, cb?: Jimp.ImageCallback): this;
         rotate(
             deg: number,
-            mode: number | boolean | undefined | null,
-            cb?: Jimp.ImageCallback<T>
+            mode: number | boolean,

```

Comment:
It seems I've made a mistake before, the type of mode should be `string | boolean` and it should be optional